tests/ciscotest.rb

```diff
@@ -176,6 +177,16 @@ def skip_nexus_i2_image?
       Utils.nexus_i2_image
   end
 
+  def system_image
+    @image ||= Platform.system_image
+  end
+
+  def defect?(pattern, msg)
+    return false unless system_image.match(Regexp.new(pattern))
+    puts "\n#{self.class}##{caller[0][/`.*'/][1..-2]} -> NOOP <- [#{msg}]"
```

Comment:
Messages should be using the logger class instead of `puts`.

Nit: I'm not a fan of printing `NOOP` or that the test is marked as passing.  During our offline discussion, did we decide on asserting defects or skipping the testcase?

ping @mikewiebe